frontend/src/components/organisms/EventForm.tsx

```diff
@@ -80,15 +234,31 @@ const EventForm = ({ eventType }: EventFormProps) => {
       </div>
       <div className="sm:space-x-4 grid grid-cols-1 sm:grid-cols-2">
         <div className="pb-4 sm:pb-0">
-          <DatePicker label="Start Date" />
+          <DatePicker
+            label="Start Date"
+            value={eventDetails ? eventDetails.startDate : undefined}
+            onChange={(e) => (e.$d != "Invalid Date" ? setStartDate(e.$d) : "")}
+          />
         </div>
-        <DatePicker label="End Date" />
+        <DatePicker
+          label="End Date"
+          value={eventDetails ? eventDetails.endDate : undefined}
+          onChange={(e) => (e.$d != "Invalid Date" ? setEndDate(e.$d) : "")}
```

Comment:
I think there should be some sort of required prop with Date and time picker too so that folks need to fill it out. 
If it proves difficult to manage required state for the date and time, you can do a quick check in the submit handler if the date and time were submitted and render an `Alert` (in atoms) if they didn't enter a date and time